layers/+tools/restclient/README.org

```diff
@@ -32,6 +32,7 @@ Also there is an [[http://emacsrocks.com/e15.html][Emacs Rocks!]] episode of it.
 |-------------+-------------------------------------------------------------|
 | ~gj~        | Jump to next request                                        |
 | ~gk~        | Jump to previous request                                    |
+| ~SPC m j~   | Jump to a variable or request in current buffer             |
```

Comment:
Should this be `SPC s j` ? In some major modes it seems similar in function but different somehow. In restclient it gives Imenu error but I know nothing about Imenu.